lib/galaxy/files/sources/onedata.py

```diff
@@ -1,30 +1,31 @@
 try:
-    from fs.onedatafs import OnedataFS
+    from fs.onedatarestfs import OnedataRESTFS
 except ImportError:
-    OnedataFS = None
+    OnedataRESTFS = None
 
-from typing import (
-    Optional,
-    Union,
-)
-
-from . import (
-    FilesSourceOptions,
-    FilesSourceProperties,
-)
+from typing import Optional
+from . import FilesSourceOptions
 from ._pyfilesystem2 import PyFilesystem2FilesSource
 
 
-class OneDataFilesSource(PyFilesystem2FilesSource):
+def remove_prefix(prefix, string):
+    if string.startswith(prefix):
+        string = string[len(prefix):]
```

Comment:
```suggestion
        string = string[len(prefix) :]
```

should fix the Python linting.
You can run `make format` from the root of your galaxy repository to fix most linter problems, see also https://github.com/galaxyproject/galaxy/blob/dev/CONTRIBUTING.md#style-guidelines